db/migrate/20170818134716_add_default_flag_to_supported_permissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultFlagToSupportedPermissions < ActiveRecord::Migration
+  def change
+    add_column :supported_permissions, :default, :boolean, default: false, null: false
```

Comment:
Lost comment due to rebase:

> @davidslv:
>> How big is this table? Adding a default value will lock the table until all records are written

> @h-lame:
>> At the time this migration is run it'll be empty because we only just created it in the previous migration.

Reply:
Originally this was a question on the changes to the `bulk_grant_permission_sets` table which is split over two commits (but ultimately only one migration).  However, it's a question that's relevant here on this commit and migration.  Unlike I suggest in my previous answer, the `supported_permissions` isn't empty and hasn't just been created, but it does have only 725 rows, so I think we're safe enough to add the column inline.